plugin/pkg/admission/limitranger/admission.go

```diff
@@ -212,6 +213,12 @@ func mergeContainerResources(container *api.Container, defaultRequirements *api.
 	for k, v := range defaultRequirements.Limits {
 		_, found := container.Resources.Limits[k]
 		if !found {
+			request, found := container.Resources.Requests[k]
```

Comment:
if the request is greater than the default limit, we should just not set the limit.

i do not see why we need to track errors.